docs/usage.rst

```diff
@@ -387,11 +387,16 @@ By extending the event with subcomponents, you can create multiple alarms.
     >>> alarm_24h_before.add('description', 'Reminder: Event in 24 hours')
     >>> event.add_component(alarm_24h_before)
 
-Or even recurrence.
+Or even recurrence, either from a dictionary or a string.
+Note that if you want to add the reccurence rule from a string, you must use the vRecur property. Otherwise the rule will be escaped, making it invalid.
```

Comment:
Add some markup for inline literal.

Use one line per sentence for easier diffs in the future.

```suggestion
Note that if you want to add the reccurence rule from a string, you must use the ``vRecur`` property.
Otherwise the rule will be escaped, making it invalid.
```